tests/runtime/test_dump_services.py

```diff
@@ -0,0 +1,72 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Standard
+import os
+import shutil
+import tempfile
+
+# First Party
+import alog
+
+# Local
+from caikit.runtime.dump_services import dump_grpc_services, dump_http_services
+import caikit.interfaces.common
+import sample_lib
```

Comment:
Initially I thought this was an extra import but then remembered we do need library information - as discussed offline could we try to use the config for sample_lib info within the tests?